app/assets/javascripts/topbar.js

```diff
@@ -9,17 +9,22 @@ $(function() {
 
 //open main menu on hover
 $(document).on('mouseenter', '.collapse .dropdown.menu_tab_dropdown', function(){
-  if(!$(this).hasClass('open')){
-    $(this).find('.dropdown-toggle:first').click();
+  var $this = $(this);
+
+  if(!$this.hasClass('open')){
```

Comment:
Isn't this check redundant? if the element already has the class it won't be added again